hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/datanodes/datanodes.tsx

```diff
@@ -389,24 +398,39 @@ export class Datanodes extends React.Component<Record<string, object>, IDatanode
     return selectedColumns;
   };
 
-  _loadData = () => {
-    this.setState(prevState => ({
-      loading: true,
-      selectedColumns: this._getSelectedColumns(prevState.selectedColumns)
-    }));
+  _loadData = async () => {
+    // Need to call decommission API on each interval to get updated status before datanode API to compare UUID's
+    // update Operation State Column in table Manually before rendering
+    let decommissionAPIRequest = this._loadDecommisionAPI();
```

Comment:
I think we don't need both `await` and `then()`

```suggestion
    let decommissionAPIResponse = await this._loadDecommisionAPI();
    decomissionUids = decommissionAPIResponse && decommissionAPIResponse.data
```

Reply:
Done changes in latest 2 commits